frontend/src/assets/tailwind.css

```
@@ -169,51 +170,83 @@ body {
 }
 
-/* Flicker Animation - Subtle and random */
+/* Flicker Animation - Subtle CRT screen flickering */
 @keyframes flicker {
-  0% {
+  0%, 100% {
     opacity: 1;
   }
+  5% {
+    opacity: 0.97;
+  }
```

Stylelint (17.2.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)